docs/configuration/tempo-config.md

```diff
@@ -226,6 +221,24 @@ tail_sampling:
         [ username: <string> ]
         [ password: <secret> ]
         [ password_file: <string> ]
+
+# Configures aggregation of spans by trace.
+# This is of particular interest for processor that benefit from having complete
+# traces, such as tail-based sampling
+#
+# A trace will be considered complete after the defined `wait` duration has
+# passed since its first span arrived to the processors.
+# If the trace is not complete by then, it'll be split into more than one trace.
+#
+# Longer waiting times will increase the number of traces that are correctly
+# grouped. However, it will also increase the memory overhead of the processor.
+group_by_trace:
```

Comment:
shouldn't the "load balancing" settings be moved out as well? maybe renamed something to suggest it's routing by trace id.